public/js/actions/sources.js

```diff
@@ -35,6 +37,11 @@ function queueSourcesDispatch(dispatch) {
  */
 function newSource(source) {
   return ({ dispatch }) => {
+    if (isEnabled("features.sourceMaps") && source.sourceMapUrl) {
+      networkRequest(source.sourceMapUrl).then(res => {
+      });
+    }
```

Comment:
agreed - i'm planning on keeping source maps in a cache for that reason, but we will talk about it soon.